python/cli/prelude_cli/views/configure.py

```diff
@@ -1,13 +1,47 @@
 import click
 
+from configparser import ConfigParser, MissingSectionHeaderError
+
 
 @click.command()
+@click.option('-f', '--file', help="path to credentials (.ini) file", default=None, type=click.Path(exists=True))
 @click.pass_obj
-def configure(account):
+def configure(account, file):
     """ Configure your local keychain """
-    profile = click.prompt('Enter the profile name', default=account.profile, show_default=True)
-    hq = click.prompt('Enter the Prelude API', default=account.hq, show_default=True)
-    account_id = click.prompt('Enter your account ID')
-    account_token = click.prompt('Enter your account token')
-    account.configure(account_id=account_id, token=account_token, hq=hq, profile=profile)
-    click.secho('Credentials saved', fg='green')
+    profile = account.profile
+    hq = account.hq
+    account_id = None
+    account_token = None
```

Comment:
You don't need to declare these, python is smart enough to know that it will get set in both cases in your if statement below